src/sdk/types.ts

```diff
@@ -171,6 +175,10 @@ export type OAuth2AuthConfig<
   display: {
     markdown: string | ((platform: Platform<{}, any, string>) => string);
   };
+  /**
+   * Scopes that will always be requested
```

Comment:
Isn't this definition the same as requiring it?

When I read defaultScopes, it feels like this is what it's defaulted if I don't provide any scopes. If I provide my own scopes, it would override these instead of extend on top.

Reply:
What about "additionalScopes" or "alwaysIncludedScopes"?

Reply:
required makes it seem like these need to be provided by the user when they make the oauth request